asv_bench/benchmarks/benchmarks.py

```diff
@@ -11,159 +11,174 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+# define `MODIN_CPUS` env var to control the number of partitions
+# it should be defined before modin.pandas import (in case of using os.environ)
+
 import modin.pandas as pd
 import numpy as np
 
-from modin.config import CpuCount, TestDatasetSize
+from modin.config import TestDatasetSize
 from .utils import generate_dataframe, RAND_LOW, RAND_HIGH, random_string
 
-# define `MODIN_CPUS` env var to control the number of partitions
-# it should be defined before modin.pandas import
-pd.DEFAULT_NPARTITIONS = CpuCount.get()
+ASV_USE_IMPL = "modin"
```

Comment:
isn't asv provides some way to set custom benchmark parameters via command-line arguments?
It would be easier just to just write:
```
asv run ASV_USE_IMPL=pandas
```
instead of overwriting variable in the source file

Reply:
I do not know about such a possibility. Where does this thought come from?

Reply:
`pytest` allows to specify custom flags in a such way, I thought maybe asv has something similar.

Anyway, if asv does not provide such functionality, we probably can specify that via some environment variable `MODIN_ASV_USE_IMPL=...`

Reply:
Got it. The documentation does not write about this possibility; yes we can use environment variable.